metadata-db-app/int-test/cmr/metadata_db/int_test/concepts/variable_association_save_test.clj

```diff
@@ -23,3 +25,54 @@
       (is (= [(str "Variable association could not be associated with provider [REG_PROV]. "
                    "Variable associations are system level entities.")]
              errors)))))
+
+(deftest save-variable-association-success-test
```

Comment:
The current implementation may not be able to handle the following two scenarios:
1. There is `&` in the value of some variable field values. e.g. variable Name is `Beauty & Beast`.
2. There are more than 2 `-d` sent in the request body.
We should add tests to show these two cases are taken care of.

Reply:
I thought the tests in core_test.clj would take care of this, but if you don't think that is enough, I'll write a new one here.

Reply:
I took another look at the code and made some changes to handle this.